woocommerce/woocommerce-android
# Issue/6547 simple payment link status

Fixes #6547 - Previously when you created a simple payment and shared the payment link, the app would mark the order as completed, which means the link would no longer work. This PR corrects this by marking the order as pending, as the iOS app does.

To test:

* Open Flipper to watch network traffic
* Create a simple payment
* On the "Take payment" screen, tap to share the payment link and copy it to the clipboard
* When you're returned to the order list, verify the `POST` to update the simple payment order set the status to pending
